ofrak_core/pytest_ofrak/patterns/unpack_modify_pack.py

```diff
@@ -20,22 +22,27 @@ class UnpackModifyPackPattern(ABC):
     """
 
     async def test_unpack_modify_pack(self, ofrak_context):
-        root_resource = await self.create_root_resource(ofrak_context)
-        await self.unpack(root_resource)
-        original_tree = await root_resource.summarize_tree()
-        await self.modify(root_resource)
-        modified_tree = await root_resource.summarize_tree()
-        print("\n")
-        print(
-            "".join(
-                ndiff(
-                    original_tree.splitlines(keepends=True), modified_tree.splitlines(keepends=True)
-                )
-            ),
-            end="",
-        )
-        await self.repack(root_resource)
-        await self.verify(root_resource)
+        try:
+            root_resource = await self.create_root_resource(ofrak_context)
+            await self.unpack(root_resource)
+            original_tree = await root_resource.summarize_tree()
+            await self.modify(root_resource)
+            modified_tree = await root_resource.summarize_tree()
+            print("\n")
+            print(
+                "".join(
+                    ndiff(
+                        original_tree.splitlines(keepends=True),
+                        modified_tree.splitlines(keepends=True),
+                    )
+                ),
+                end="",
+            )
+            await self.repack(root_resource)
+            await self.verify(root_resource)
+        except CalledProcessError as e:
+            # Better printing of errors if something goes wrong in test setup/execution
+            ComponentSubprocessError(e)
```

Comment:
Is this needed? I thought the component itself already raises a `ComponentSubprocessError`.

Reply:
A lot of the tests ran their own subprocess calls to setup the test files, with the common pattern that they catch and pretty-print the error. I debated just removing that catch entirely - those should never fail, these are the commands to set up the ground truth test assets - but it would help with debugging, so I added the catch here that takes care of the pretty-printing.